### ISSUE TITLE
[ADE 508 Audit]: VA Notify Portal: 508 Audit Report for 43660 VA Notify Portal - Defect 1 - Critical

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#109083]
> [!NOTE]
> **QUICK SUMMARY:** Child ticket of #109082  to track the work related to Defect 1 - Critical. 

### Defect 1 | Critical | Created:
_WCAG 1.1.1 Non-text Content | All non-text content that is presented to the user has a text alternative that serves the equivalent purpose, except for the situations listed below._

Throughout the product, images do not have sufficient text equivalents. For example, on the VA Notify screen, under Discover the VA Notify experience, “Safety and security Graphic” is read for the first image. If these images are determined to be decorative, they should be implemented to be ignored by assistive technology. (New: 03/25/2025)

### Feedback from Project Team
Throughout the product, images do not have sufficient text equivalents. For example, on the VA Notify screen, under Discover the VA Notify experience, “Safety and security Graphic” is read for the first image. If these images are determined to be decorative, they should be implemented to be ignored by assistive technology. (New: 03/25/2025)

[staging.notifications.va.gov](http://staging.notifications.va.gov/) (home page)
(A fix for this issue is in place)

- ![Image](https://github.com/user-attachments/assets/b452f786-241b-469f-a3ae-67e15b51b0dd)

### Ticket Updates
- [ ] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [ ] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._
